include/NotePlayHandle.h

```diff
@@ -258,6 +258,11 @@ class EXPORT NotePlayHandle : public PlayHandle, public Note
 		m_frequencyNeedsUpdate = true;
 	}
 
+	bool isReleaseStarted() const
+	{
+		return m_releaseStarted;
+	}
```

Comment:
you stick this to the end of `NotePlayHandle` but it belongs, logically, together with `isReleased()`.
Please move `isReleaseStarted()` to after `isReleased()` and `m_releaseStarted` to after `m_released`.